bigquery/datasets.js

```diff
@@ -15,123 +15,117 @@
 
 'use strict';
 
-const BigQuery = require('@google-cloud/bigquery');
+function createDataset (datasetId, projectId) {
+  // [START bigquery_create_dataset]
+  // Imports the Google Cloud client library
+  const BigQuery = require('@google-cloud/bigquery');
+
+  // The project ID to use, e.g. "your-project-id"
+  // const projectId = "your-project-id";
```

Comment:
Standards question: should we consistently specify project IDs in samples, or not?

(I don't think we enforce this particular standard yet, but I could be wrong...)